src/ext/framework/command.rs

```diff
@@ -14,6 +14,12 @@ pub type AfterHook = Fn(&Context, &Message, &String, Result<(), String>) + Send
 pub type InternalCommand = Arc<Command>;
 pub type PrefixCheck = Fn(&Context) -> Option<String> + Send + Sync + 'static;
 
+#[doc(hidden)]
+pub enum CommandKind {
```

Comment:
This might need a different name, having both `CommandKind` and `CommandType` is kind of meh.